pkg/webhook/bootstrap.go

```diff
@@ -129,6 +135,14 @@ func (s *Server) installWebhookConfig() error {
 		return s.err
 	}
 
+	if !s.GenerateManifests && s.DisableInstaller {
+		log.Info("webhook installer is disabled")
```

Comment:
log seems to be incorrect. may be add "and generating webhook manifests is also disabled"